php-plugin/src/main/java/fr/greencodeinitiative/php/PhpRuleRepository.java

```diff
@@ -38,48 +29,59 @@
 import fr.greencodeinitiative.php.checks.IncrementCheck;
 import fr.greencodeinitiative.php.checks.NoFunctionCallWhenDeclaringForLoop;
 import fr.greencodeinitiative.php.checks.UseOfMethodsForBasicOperations;
-import org.sonar.api.rules.RuleType;
+import org.sonar.api.SonarEdition;
+import org.sonar.api.SonarProduct;
+import org.sonar.api.SonarQubeSide;
+import org.sonar.api.SonarRuntime;
 import org.sonar.api.server.rule.RulesDefinition;
-import org.sonar.api.server.rule.RulesDefinitionAnnotationLoader;
+import org.sonar.api.utils.Version;
 import org.sonar.plugins.php.api.visitors.PHPCustomRuleRepository;
+import org.sonarsource.analyzer.commons.RuleMetadataLoader;
 
-public class PhpRuleRepository implements RulesDefinition, PHPCustomRuleRepository {
```

Comment:
You added a PhpRulesRepository (same for python) file that's really close to the name of this one. Maybe rename one of the two files ?